Classes/Service/AuthenticationService.php

```diff
@@ -375,10 +376,8 @@ protected function convertResourceOwner(array $info): bool|array
             ]
         );
 
-        // preserve username and password for existing users
+        // preserve password for existing users, an existing password should not be replaced on every login
         if ($row) {
```

Comment:
It might actually be nicer to invoke the password generation only for new users. 

With the suggested implementation in this PR, the password will first be generated and then unset.

Reply:
Yes, good one. Will add this.